libcore/src/IO/GeoFileParser.cpp

```diff
@@ -837,10 +837,11 @@ Goal * GeoFileParser::parseWaitingAreaNode(TiXmlElement * e)
     // Additional checks:
     const bool waitPed =
         (wa->GetMinNumPed() > 0 && wa->GetMaxNumPed() > 0 && wa->GetWaitingTime() >= 0.);
-    const bool waitDoor = (wa->GetTransitionID() > 0);
+    const bool waitDoor   = (wa->GetTransitionID() > 0);
+    const bool waitGlobal = (wa->IsGlobalTimer() && wa->GetWaitingTime() >= 0.);
 
     // Either (minPed, maxPed, waitingTime) OR transitionID are set
-    if(!waitPed && !waitDoor) {
+    if(!waitPed && !waitDoor && !waitGlobal) {
```

Comment:
The comment above does not fit the code. It should probably just be removed.

Reply:
comment removed